python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -522,13 +382,16 @@ def set_params(self, **params):
         Parameters
         -----------
         params : dict of new params
+        worker_numb : list (default = None)
+            If worker_numb is `None`, then the parameters will be set for all
+            the workers. If it is not `None` then a list of worker numbers
+            for whom the model parameter values have to be set should be
+            passed.
+            ex. worker_numb = [0], will only update the parameters for
```

Comment:
I think the worker_numb part seems error-prone for users, and I'd love to eliminate it if possible or thoroughly hide to from users.